templates/mariadb.service.j2

```diff
@@ -35,7 +35,7 @@ Type=notify
 
 # Setting this to true can break replication and the Type=notify settings
 # See also bind-address mysqld option.
-PrivateNetwork=false
+PrivateNetwork={{ mysql_private_network | default('false') }}
```

Comment:
In order to provide a better discoverability of this vars, I think it would be nice to add that vars with a default value in default vars file even if default('value') filter is used here

Reply:
Ok, done!